process/block/shardblock.go

```diff
@@ -258,6 +258,14 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	if header.IsStartOfEpochBlock() {
```

Comment:
Do you need this part of code also in commit block again?

Reply:
yes. As the leader cannot make revert in create or applybodytoheader.

Reply:
ok